### PR TITLE
Header Changes

### PR DESCRIPTION
**Pull request by sbhaktha**
@codeviking : quick review please!

screenshot:

screen shot 2016-06-01 at 9 46 11 pm

### CODE REVIEW

File: webapp/app/css/_header.less [hunk 1]
```diff
@@ -20,4 +20,12 @@ header {
       width: 24px;
     }
   }
-}
\ No newline at end of file
+
+  .header1 {
+    margin: 0 0 0 0;
```
Comment (codeviking) [inline webapp/app/css/_header.less]: You can shorthand this as `margin: 0`.

File: webapp/app/css/_header.less [hunk 2]
```diff
@@ -20,4 +20,12 @@ header {
...
+  }
+
+  .header2 {
+    margin: 0 0 16px 0;
+  }
```
Comment (codeviking) [inline webapp/app/css/_header.less]: I'd just add padding to the bottom of the `header` tag and use divs (without any default styles) to create breaks:

```
<div>IKE, Getting Started</div>
<div>Learn more...</div>
```

```
header {
   padding-bottom: 16px;
}
```

**Comment by codeviking** [comment]
One nit and one suggestion -- merge when you fee it's g2g!

**Comment by sbhaktha** [comment]
good idea, done!

**Comment by sbhaktha** [comment]
will merge on green

**Pull request merged**